content/en/llm_observability/sdk.md

```diff
@@ -104,6 +104,18 @@ LLMObs.enable(
 : optional - _string_
 <br />The name of the service used for your application. If not provided, this defaults to the value of `DD_SERVICE`.
 
+### Serverless setup
+
+Enable LLM Observability in a serverless environmant by using the [Datadog-Python and Datadog-Extension layers][14].
+
+In order for these serverless layers to properly instrument and flush traces from your serverless application, specify `DD_LLMOBS_ENABLED` in your environment. This applies to both manual and auto-instrumented use cases.
+
+#### Flushing in serverless environments
```

Comment:
IMO we can completely remove this part from the documentation or create a more hidden FAQ section since a user _should_ never have to do this unless something is wrong on our end